README.md

```diff
@@ -80,6 +80,8 @@ make proto-gen
 
 The canonical branches in this repo are based on CometBFT releases. For example: [`v0.34.x-celestia`](https://github.com/celestiaorg/celestia-core/tree/v0.34.x-celestia) is based on the CometBFT `v0.34.x` release branch and contains Celestia-specific changes.
```

Comment:
> It would actually be more accurate to call the versions based off the abci version

@cmwaters are you proposing an edit to this line?

Reply:
no I think this is fine as it stands. It was more a thought said out loud